src/components/new-request/search.vue

```diff
@@ -288,12 +312,13 @@ import { AccountType, CompanyType, EntityType, Location, NrRequestActionCodes, N
 import { CommonMixin } from '@/mixins'
 import { CanJurisdictions, ConversionTypes, Designations, IntlJurisdictions, RequestActions } from '@/list-data'
 import { GetFeatureFlag } from '@/plugins'
+import BusinessLookupServices from '@/services/business-lookup-services'
 
 /**
  * This is the component that displays the new NR menus and flows.
  */
 @Component({
-  components: { BulletsColinLink, NameInput }
+  components: { BulletsColinLink, NameInput, BusinessLookup }
```

Comment:
Would it be possible to alphabetize these lines please? Also lines 21 and 22 in interfaces/index.ts. 

It's a small change but it'll keep the code clean for future readability for other devs. Thanks Kevin!

Reply:
I did

Reply:
Awesome, thanks. Resolving.